setup.py

```diff
@@ -129,26 +155,239 @@ def build_extension(self, ext):
             raise BuildExtFailed()
 
 
-macros = []
-if sys.byteorder == "big":
-    macros = [("__BIG_ENDIAN__", "1")]
-else:
-    macros = [("__LITTLE_ENDIAN__", "1")]
+def get_msgpack_extensions():
```

Comment:
Just an idea: could we copy the `setup.py` file from `psutil` and import it its function here rather than copy/pasting?

Reply:
That was something I was interested in trying, but can we register a sub-module via `setup()`? (I haven't tested yet, been meaning to)

Reply:
If by register you mean `import`, AFAIK you can as long as it's a sub-directory of your `setup.py`. So you could `import ddtrace.vendor.psutil.setup` in `setup.py`.

Reply:
if we did that, we'd probably have to rename the package and access by some name other than `ddtrace.vendor.psutil`?

Reply:
Hmmm why?

Reply:
we talked in Slack quick to align on thoughts here.

we can't import `psutil`s `setup.py` directly because it calls `setup()` which isn't what we want.

however, we can try to extract the gnarly c-extension specific things into a different module to make our `setup.py` much cleaner.

Reply:
Gave this a try on the latest version.